keysign/gpgmks.py

```diff
@@ -316,6 +316,8 @@ def sign_keydata(keydata, error_cb=None, homedir=None):
 
 
         for uid in uidlist:
+            # uid needs to be unescaped or the export will fail if it contains characters like colon.
```

Comment:
cool. Can you make it a function so that we have the decoding in only one place rather than two..?
Also, include in the comment that it should be Monkeysign's responsibility to give us the actual  bytes, not the encoded ones.  It's probably also worth including that we're only guessing the encoding without having really looked into how gpg encodes data.